typos.toml

```diff
@@ -0,0 +1,96 @@
+# Hello and Welcome to typos.toml!
+
+## Policy
+# tldr: Don't let CI bully you. Use whatever varaible names you want.
+#
+# The spell check CI feature exists to help detect accidental typos, not to constrain the
+# language we use in this repo. If you want to use a word or a variable name or somesuch
+# that the CI check is not happy with, go right ahead, just make sure to add the appropriate
+# entry to this file to tell the CI check that you didn't make a mistake. As a reviewier, if
+# you think that a variable name is bad, go right ahead and tell the author that, just don't
+# cite the CI check as a reason, that's not what the check is for.
+
+## How to
+# tldr: Add the whole identifier right below [default.extend-identifiers]
+#
+# There are two magor sections in this file, [default.extend-identifiers] (near the top) and
+# [default.extend-words] (near the bottom). If the false positive is a full identifier like
+# `mrs_claus` or `breal`, add it to [default.extend-identifiers] and that specific
+# identifier will no longer be flagged as a typo. If, on the other hand, it is a word that
+# appeasrs inside multiple a multi-word identifiers, like `ser` which is used in
+# `jl_lookup_ser_tag`, `ser_tag`, `ser_version`, etc. then insrted of adding each of those
+# identifiers individually, simply add it to the [default.extend-words] section. Identifiers
+# are case sensitive and words are not.
+
+[default.extend-identifiers]
+mrs_claus = "mrs_claus"
+womens = "womens"
+breal = "breal"
+Breal = "Breal"
+Numer = "Numer"
+splitted = "splitted"
+affinitized = "affinitized"
+key_smove = "key_smove"
+readed_zero = "readed_zero"
+check_inconsistentcy = "check_inconsistentcy"
+L_OP_CALLL = "L_OP_CALLL"
+OP_CALLL = "OP_CALLL"
+
+# strings in test directories
+Facilisi = "Facilisi"
+vailable = "vailable"
+
+# typos that are programmatically observable
+supress_output = "supress_output"
+hashs_seed = "hashs_seed"
+
+[default.extend-words]
+ba = "ba"
+egal = "egal"
+parm = "parm"
+modul = "modul"
+nd = "nd"
+strat = "strat"
+WRONLY = "WRONLY"
+thisy = "thisy"
+vally = "vally"
+Ot = "Ot"
+clos = "clos"
+applys = "applys"
+findn = "findn"
+mis = "mis"
+seh = "seh"
+ue = "ue"
+Missings = "Missings"
+SOM = "SOM"
+sais = "sais"
+Merly = "Merly"
+Filetimes = "Filetimes"
+HSA = "HSA"
+Strategems = "Strategems"
+anumber = "anumber"
+uupper = "uupper"
+shttp = "shttp"
+ser = "ser"
+somes = "somes"
+sav = "sav"
+egals = "egals"
+OLT = "OLT"
+eyt = "eyt"
+matc = "matc"
+nam = "nam"
+alls = "alls"
+noe = "noe"
+
+# strings in tests directories
+fo = "fo"
+Uest = "Uest"
+Dows = "Dows"
```

Comment:
These should be in `extend-ignore-identifiers-re` with some context.
```toml
extend-ignore-identifiers-re = [
    "^[a-zA-Z][a-zA-Z]?[a-zA-Z]?[a-zA-Z]?$",
    '"QU QUer QUest"',
    '"wiNDows"',
]
```

Reply:
I really just want to do [`extend-words-re = "^[a-zA-Z][a-zA-Z]?[a-zA-Z]?[a-zA-Z]?$"`](https://github.com/crate-ci/typos/issues/852) and delete all of the short entries in extend-words.